tests/test_api.py

```diff
@@ -13,7 +13,8 @@ def test_simple(tmpdir):
             output_dir="output",
             map_exe=TESTDATA_DIR/"word_count/map.py",
             reduce_exe=TESTDATA_DIR/"word_count/reduce.py",
-            num_reducers=4
+            num_reducers=4,
+            partitioner=None,
```

Comment:
I think `partioner` has a default value of `None`.  At least one test should use the default value.

Reply:
Are you talking about the function definition of the `mapreduce()` function?

```
def mapreduce(
    input_path,
    output_dir,
    map_exe,
    reduce_exe,
    num_reducers,
    partitioner,
):
```
None of the other optional arguments has a default value here, so I thought `partitioner` should be consistent.

Reply:
Ah, I mixed those up.  I might consider the `partitioner` input a rarely used and optional input, so maybe a default value would make sense in the `mapreduce()` function.

Reply:
Done 11e42f1878887261e4d7ddc47ceb5449172022d2